### ISSUE TITLE
Create page for describing "graph-ready data"

### ISSUE THREAD

**Issue by alyssadai** [neurobagel/documentation#32]
We currently have a Data Dictionaries page that describes and links to examples of the output of the annotation tool.

However, it would be helpful (for both users + devs) to also document the final .jsonld (i.e., output of CLI) that represents data ready to be uploaded to a graph. This doesn't fit on the Data Dictionaries page, in my opinion.

We could create a new page where we briefly describe the difference between the data dictionary and a .jsonld file (I guess we could call these "subject-level annotations" or something?) and then link to the phenotypic [example_synthetic.jsonld](https://github.com/neurobagel/bagel-cli/blob/main/bagel/tests/data/example_synthetic.jsonld) as well as the phenotypic + imaging [example_synthetic_pheno-bids.jsonld]()

As part of this issue, we will also reorder the docs pages as follows:
```
  - User guide
    - Overview
    - Preparing data for annotation
    - Annotating a dataset
    - Generating harmonized subject-level metadata
    - Setting up a graph
    - Using the API
    - Running cohort queries
  - Data models
    - Augmented BIDS data dictionaries
    - Data files for Neurobagel graphs
    - Naming conventions for terms
  - Preparing data using mr_proc
    - ...
  - Our team
  - Getting help
  - Citing Neurobagel
```

**Comment by surchs** [comment]
Agree, I think this also links to JBs question about clarifying terminology. To me this would be about the data model, and the existing section about the data dictionary

**Comment by alyssadai** [comment]
Would it make sense to have an umbrella "Data schema(s)" section which includes two pages, one for the "Data dictionary for tabular data columns" (i.e., the `json`) and one for "Modelling participant-level properties" (i.e., the `jsonld`) or something like that?

In my view the current [Data dictionaries](**url**) page describes our internal schema for the data dictionary/how we model the meaning of specific columns in a TSV, and the page we want to add (as part of this issue) describes a similar schema for the output jsonld. I think I'm probably conflating what a "data schema" vs "data model" is, but maybe it makes sense to pick one term to describe the format of Neurobagel-specific files generally, and then talk about each file in more detail. 

In other words, to me we have a data model for **both** the data dictionary (`dictionary_models.py` in the CLI) and the jsonld (`models.py`). To distinguish these I think we could either use a different term for each (e.g., "data dictionary schema" vs. "data model") or just use one term and then talk about specifics of each file (kind of like the [BIDS docs](https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#participants-file), which just describes things at the level of specific .json files). 

Does that make any sense?

**Comment by surchs** [comment]
I think that makes sense to me. So the structure would be:

- Data Models
  - for augmented BIDS data dictionaries
  - for Neurobagel Graph
?

**Comment by surchs** [comment]
Demoting because not linked to OHBM/OpenNeuro priority - please ping here if you disagree

**Comment by surchs** [comment]
hey @alyssadai: in prep for our planned reorganization of pages into new sections, how about we repurpose this issue to achieve that. I.e.:

- Getting Started
  - tool1
  - tool2
- Data models
  - data dictionaries
  - graph model
  - naming conventions
- Data preparation
  - all of the mr_proc stuff
- How to cite

And remove
- welcome
- getting started
- About us

As those will be handled by the landing page. Makes sense?

**Comment by alyssadai** [comment]
Just to clarify the plan, since the new "Getting started" page currently includes an overview of the ecosystem tools (which I believe we want to keep?), is the intended change to go from this (where the top-level bullets would be the tabs of the website):

```
  - Welcome
  - Getting started
  - User guide
    - Preparing data for annotation
    - Annotating a dataset
    - Generating harmonized subject-level metadata
    - Setting up a graph
    - Using the API
    - Running cohort queries
  - Data models
    - Augmented BIDS data dictionaries
    - Data files for Neurobagel graphs
  - Naming conventions for terms
  - Preparing data using mr_proc
    - ...
  - About us
  - Citing Neurobagel
```

to this:

**Comment by alyssadai** [comment]
Just to clarify the plan, since the new "Getting started" page (https://github.com/neurobagel/documentation/blob/adai/add-tools-models-pages/docs/start.md) currently includes an overview of the ecosystem tools (which IIRC we want to keep?), is the intended change to go from this (where the top-level bullets would be the tabs of the website):

```
  - Welcome
  - Getting started
  - User guide
    - Preparing data for annotation
    - Annotating a dataset
    - Generating harmonized subject-level metadata
    - Setting up a graph
    - Using the API
    - Running cohort queries
  - Data models
    - Augmented BIDS data dictionaries
    - Data files for Neurobagel graphs
  - Naming conventions for terms
  - Preparing data using mr_proc
    - ...
  - About us
  - Citing Neurobagel
```

to something like this:
```
  - User guide
    - Ecosystem overview  # from Getting Started
    - Preparing data for annotation
    - Annotating a dataset
    - Generating harmonized subject-level metadata
    - Setting up a graph
    - Using the API
    - Running cohort queries
  - Data models
    - Augmented BIDS data dictionaries
    - Data files for Neurobagel graphs
    - Naming conventions for terms
  - Preparing data using mr_proc
    - ...
  - Citing Neurobagel
```

And I'm assuming we want the "User guide" section in the second layout to be renamed to "Getting started", right?

**Comment by surchs** [comment]
@alyssadai , the only requirement is that every top level section (I.e. future tab) should have at least two sub pages because with the move to tabs the left hand nav changed its purpose. 

What exactly the sections are is less important as long as they are generally intuitive for a user to navigate the docs.

**Comment by alyssadai** [comment]
Gotcha, will do 👍
